### PR TITLE
Add various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Added various typos

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -23984,6 +23985,7 @@ extradiction->extradition
 extraenous->extraneous
 extranous->extraneous
 extraordinarly->extraordinarily
+extrapoled->extrapolated
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:23988]: Other (perhaps all?) verbal forms too?
```suggestion
extrapoled->extrapolated
extrapoling->extrapolating
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -54926,6 +54936,7 @@ windos->windows
 windowz->windows
 windwo->window
 windwos->windows
+winge->wing, whine,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:54939]:
```suggestion
winge->wing, winged, whine, winger, twinge, singe, wince,
```
Perhaps even `binge`, `hinge`, `minge`, `tinge`, `wine`?

**Comment by DimitriPapadopoulos** [comment]
@luzpaz Please integrate #3080 to fix the CI errors.

**Comment by luzpaz** [comment]
@DimitriPapadopoulos I can't ATM. Please cherrypick this PR in to another PR if you don't mind ?

**Comment by DimitriPapadopoulos** [comment]
Can you at least approve #3080 for me? Then I'll rebase your branch.

**Comment by DimitriPapadopoulos** [comment]
I have rebased, CI tests pass now.

@luzpaz Are you able to have a look at my suggestions?

**Comment by luzpaz** [comment]
@DimitriPapadopoulos yes, tries to merge them via GH GUI and ran in to that problem we encountered recently due to file size of the dictionary.txt being too large.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**